content/kubermatic/main/architecture/supported-providers/aws/_index.en.md

```diff
@@ -60,6 +60,7 @@ Ensure that the assigned policy contains at least the following permissions. Pol
             "Effect": "Allow",
             "Action": [
                 "ec2:*",
+                "iam:TagInstanceProfile",
```

Comment:
Looking at the permission in question, I think this should only be given for `"Resource": "arn:aws:iam::YOUR_ACCOUNT_ID:instance-profile/*"`, i.e. the first block of permissions. So I would move this to line 30.

Reply:
hmm I would need to test this (will do after kubecon)

Reply:
Did this ever happen? 😁

Reply:
changed it and tested it finally now :-P So it wokrs